docs/cep/query-guide/functions/sourcemapper/csv.md

```diff
@@ -2,42 +2,43 @@
+| Name    | Description  | Default Value | Possible Data Types | Optional | Dynamic |
 |---------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------|---------------------|----------|---------|
-| delimiter                 | When converting a CSV format message to Stream App event, this parameter indicatesinput CSV message's data should be split by this parameter                                                                                                                                                                                                              | ,             | STRING              | Yes      | No      |
-| header.present            | When converting a CSV format message to Stream App event, this parameter indicates whether CSV message has header or not. This can either have value true or false.If it's set to `false` then it indicates that CSV message has't header.                                                                                                               | false         | BOOL                | Yes      | No      |
-| fail.on.unknown.attribute | This parameter specifies how unknown attributes should be handled. If it's set to `true` and one or more attributes don't havevalues, then SP will drop that message. If this parameter is set to `false`, the Stream Processor adds the required attribute's values to such events with a null value and the event is converted to a Stream App event. | true          | BOOL                | Yes      | No      |
-| event.grouping.enabled    | This parameter specifies whether event grouping is enabled or not. To receive a group of events together and generate multiple events, this parameter must be set to `true`.                                                                                                                                                                           | false         | BOOL                | Yes      | No      |
+| delimiter                 | When converting a CSV format message to a stream worker event, this parameter indicates input CSV message's data should be split by this parameter.   | ,             | STRING              | Yes      | No      |
+| header.present            | When converting a CSV format message to Stream App event, this parameter indicates whether CSV message has header or not. This can either have value `true` or `false`. If it's set to `false`, then it indicates that CSV message has no header.   | false         | BOOL                | Yes      | No      |
```

Comment:
@StoyanTc and @ashish-macrometa - This parameter was not present in the reference version of the doc. Is it correct?

Reply:
@diana-macrometa - I guess you mean `header.present`? Yes, it's valid.